### ISSUE TITLE
Show/Sort on Fur type

### ISSUE THREAD

**Issue by MrAndy42** [kk49/deca_website#12]
For us sad people who like to collect all the different fur types.

**Comment by wikusasdgmail** [comment]
Please if it is possible. .

**Comment by Swooz777** [comment]
Me too as I can't find any other tool online for this.

**Comment by ante7906** [comment]
Animal Population Scanner now has this possibility but not showing where the animals are, just if they are present on the map. Maybe a collaboration with that team could fix this.

**Comment by barryb777** [comment]
Yes, you can hunt some of them if you use both tools (Variable amount of groups you need to check out, kinda exciting.) The impossible one are the furs on loners, no way you can find those because animals respawn live.
